fabcar/startFabric.sh

```diff
@@ -11,9 +11,9 @@ set -e
 export MSYS_NO_PATHCONV=1
 starttime=$(date +%s)
 CC_SRC_LANGUAGE=${1:-"go"}
-CC_SRC_LANGUAGE=`echo "$CC_SRC_LANGUAGE" | tr [:upper:] [:lower:]`
+CC_SRC_LANGUAGE=$(echo "$CC_SRC_LANGUAGE" | tr '[:upper:]' '[:lower:]')
 
-if [ "$CC_SRC_LANGUAGE" = "go" -o "$CC_SRC_LANGUAGE" = "golang" ] ; then
+if [ "$CC_SRC_LANGUAGE" = "go" ] -o [ "$CC_SRC_LANGUAGE" = "golang" ] ; then
```

Comment:
This syntax is actually incorrect and fails:
$ ./startFabric.sh 
./startFabric.sh: line 16: [: too many arguments
The following will work:
```suggestion
if [ "$CC_SRC_LANGUAGE" = "go" ] || [ "$CC_SRC_LANGUAGE" = "golang" ] ; then
```